plutus-core/plutus-core/src/PlutusCore/Evaluation/Result.hs

```diff
@@ -21,8 +21,6 @@ import PlutusCore.Pretty
 import Control.Monad.Except (MonadError, catchError, throwError)
 
 -- | The parameterized type of results various evaluation engines return.
--- On the PLC side this becomes (via @makeKnown@) either a call to 'Error' or
--- a value of the PLC counterpart of type @a@.
```

Comment:
I disabled `makeKnown` for `EvaluationSuccess` a long time ago, so this comment has been a lie for quite a while.